src/components/pages/search-pages/SearchPage.styles.ts

```diff
@@ -0,0 +1,108 @@
+import { SxProps } from '@mui/material/styles';
+import { Theme } from '@mui/system';
+
+export const layout: SxProps<Theme> = {
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'column',
+  padding: {
+    tablet: '12px 16px',
+    desktop: '0px 80px 32px 80px',
+  },
+  columnGap: {
+    desktop: 'auto',
+    mobile: '15px',
+  },
+  marginBottom: {
+    desktop: 'auto',
+    mobile: '24px',
+  },
+};
+
+export const form: SxProps<Theme> = {
+  display: 'grid',
+  gridTemplateRows: '1fr',
+  gap: '12px',
+  marginBottom: '34px',
+  alignItems: 'center',
+  gridTemplateColumns: {
+    tablet: '35% 45% 10%',
+    desktop: '35% 15% 15% 5%',
+  },
+  justifyContent: {
+    tablet: 'center',
+    desktop: 'auto',
+  },
+};
+
+export const collapseBtn: SxProps<Theme> = {
```

Comment:
if there is only `display: none`, why don't you remove this button from page?

Reply:
![image](https://github.com/fictadvisor/fictadvisor-web/assets/68693183/ae196c9b-5143-405f-80a8-41cf71df2a17)